src/components/NavLinks/SideBarLinks.js

```diff
@@ -78,7 +72,8 @@ const SelectedListItem = ({ items }) => {
   };
 
 SelectedListItem.propTypes = {
-  items: PropTypes.array.isRequired
+  items: PropTypes.arrayOf(PropTypes.array),
```

Comment:
I think this is not properly defined, if it's a matix then PropTypes.arrayOf(PropTypes.arrayOf(Proptypes.shape({something})))?